application/common/tests/test_dynamodb.py

```diff
@@ -240,35 +251,15 @@ def copy_and_modify_website(ce, new_website: str):
     return copy
 
 
-def test_get_newest_event_per_odscode(dynamodb_table_create, change_event, dynamodb_client, dynamodb_resource):
```

Comment:
The test for function mentioned in the other comment, should it be commented out?